### PR TITLE
added ProductHunt votes count, notification bell on change

### PR DESCRIPTION
**Pull request by softvar**

### CODE REVIEW

File: Web/ProductHunt/producthunt-posts-votes.1h.sh [hunk 1]
```diff
@@ -0,0 +1,159 @@
+#!/usr/local/bin/node
+
```
Comment (keithamus) [inline Web/ProductHunt/producthunt-posts-votes.1h.sh]: Hey @softvar - do you want to add some metadata please, check out https://github.com/matryer/bitbar#metadata for more. You'll need to at least add the title, version, description & image

Comment (softvar) [inline Web/ProductHunt/producthunt-posts-votes.1h.sh]: Hi @keithamus 
It's done :)

Comment (keithamus) [inline Web/ProductHunt/producthunt-posts-votes.1h.sh]: Same problem as the other commit btw

Comment (softvar) [inline Web/ProductHunt/producthunt-posts-votes.1h.sh]: My mistake, fixed now.

**Pull request merged**